services/wallet/activity/activity.go

```diff
@@ -0,0 +1,313 @@
+package activity
```

Comment:
If I understand correctly, this package is the query side and the transfer package is the collector?